src/vl-core.js

```diff
@@ -23,8 +23,17 @@ export const VlElement = (SuperClass) => {
             }
         }
 
+        /**
+         * Geeft de prefix die gebruikt kan worden voor attributen.
+         * 
+         * @returns {String} attribuut prefix
+         */
+        static get attributePrefix() {
```

Comment:
Met dat er verwezen wordt naar `VlElement.attributePrefix` gaat deze prefix wel nooit overschreven kunnen worden door een afnemer. 
Ik kan nu niet op 1 2 3 op use cases komen waar dit problemen gaat geven, of waar je zou willen afwijken van "data-vl-", dus misschien is het effectief ook geen issue.

Reply:
Ik kan ook geen use cases bedenken, we dwingen al de webcomponenten (op termijn) om specifiek deze prefix te gebruiken.